From c6d4b71ac137193310a220e2c6935fb4635c2a3f Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Wed, 1 Jul 2020 19:12:50 +0100
Subject: [PATCH] Add LIBDIR workaround for GNUInstallDirs

---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 350d6263..5f984578 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,6 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 3.0.2)
 PROJECT( octomap )
+set(CMAKE_INSTALL_LIBDIR "lib") # workaround for GNUInstallDirs
 
 include(CTest)
 include(GNUInstallDirs)
-- 
2.25.1

