From e94d78fa48d165a3155eb178cb0fd909aacbb61d Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Sun, 29 Mar 2020 02:08:02 +0100
Subject: [PATCH 1/2] Add workaround for libdir

---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 974fbab..1ac0639 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -55,6 +55,8 @@ INCLUDE_DIRECTORIES(BEFORE SYSTEM ${OCTOMAP_INCLUDE_DIRS})
 # (this registers the build-tree with a global CMake-registry)
 export(PACKAGE octovis)
 
+set(CMAKE_INSTALL_LIBDIR "lib/")
+
 set(INSTALL_TARGETS_DEFAULT_ARGS
   RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}"
   LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
-- 
2.17.1

