From 28d10436dd0d3fc95cc63ade0b0f03388e95061c Mon Sep 17 00:00:00 2001
From: William Woodall <william@osrfoundation.org>
Date: Wed, 24 Apr 2013 13:44:44 -0700
Subject: [PATCH 2/2] Fix installation location of package.xml

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1113df9..c85b505 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -117,7 +117,7 @@ IF(BUILD_VIEWER)
   install(FILES ${octovis_HDRS} DESTINATION include/octovis)
 
   # Install catkin package.xml
-  install(FILES package.xml DESTINATION share/octomap)
+  install(FILES package.xml DESTINATION share/octovis)
   
 ELSE()
     MESSAGE ( "Unfortunately, the viewer (octovis) can not be built because some requirements are missing.")
-- 
1.7.12.4 (Apple Git-37)

